src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+class Rectangle extends Figure {
+    private double length;
+    private double width;
+
+    public Rectangle(String color, double length, double width) {
+        super(color);
+        this.length = length;
+        this.width = width;
+    }
+
+    @Override
+    public double getArea() {
+        return length * width;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle, area: "
+                + getArea() + " sq. units, length: "
+                + this.length + ", width: " + this.width + " units, color: " + this.color);
```

Comment:
Consider using `getColor()` instead of directly accessing the `color` field to maintain encapsulation and adhere to OOP principles.